### PR TITLE
refactor: clean up code

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

File: internal/database/offline/sqlutil/join.go [hunk 1]
```diff
@@ -316,8 +316,8 @@ func sqlxQueryResults(ctx context.Context, dbOpt dbutil.DBOpt, query string, hea
 				}
 			}
 
-			defer rows.Close()
-			defer close(data)
+			rows.Close()
+			close(data)
```
Comment (wfxr) [inline internal/database/offline/sqlutil/join.go:320]: Is it ok to change the close order?

**Comment by wfxr** [comment]
suggested commit message: `refactor: clean up code`

**Comment by lianxmfor** [comment]
> suggested commit message: `refactor: clean up code`

done

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**